src/main/java/com/coveros/selenified/utilities/ReportOverview.java

```diff
@@ -0,0 +1,297 @@
+        writeStylesheet();
+        writer.println("    </head>");
+    }
+
+    /**
+     * Sets the css and js for the test report
+     */
+    @Override
+    protected void writeStylesheet() {
+        writer.println("        <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css'>\n");
+        writer.println("        <script src='https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js'></script>\n");
+        writer.println("        <script src='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js'></script>\n");
+        writer.println("        <link rel='stylesheet' href='https://cdn.datatables.net/1.10.19/css/jquery.dataTables.min.css'>\n");
+        writer.println("        <script src='https:////cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js'></script>\n");
+        writer.println("        <script>\n");
+        writer.println("            $(document).ready( function () {\n");
+        writer.println("                $('.sort').DataTable();\n");
+        writer.println("            } );\n");
+        writer.println("        </script>\n");
+    }
+
+    /**
+     * Create the title and content of the test report
+     */
+    @Override
+    protected void writeBody() {
+        writer.println("    <body>");
+        writer.println("        <h1 style='text-align:center;'>Selenified Test Results</h1>");
+        writeSuiteSummary();
+        writeScenarioDetails();
+        writer.println("    </body>");
+    }
+
+    /**
+     * Generates the high level summary of the test report
+     */
+    @Override
+    protected void writeSuiteSummary() {
+        int totalPassedTests = 0;
+        int totalSkippedTests = 0;
+        int totalFailedTests = 0;
+        long totalDuration = 0;
+
+        for (SuiteResult suiteResult : suiteResults) {
+            for (TestResult testResult : suiteResult.getTestResults()) {
+                totalPassedTests += testResult.getPassedTestCount();
+                totalSkippedTests += testResult.getSkippedTestCount();
+                totalFailedTests += testResult.getFailedTestCount();
+                totalDuration += testResult.getDuration();
+            }
+        }
+        String overallCssClass = "success";
+        if (totalSkippedTests > 0) {
+            overallCssClass = "warning";
+        }
+        if (totalFailedTests > 0) {
+            overallCssClass = "danger";
+        }
+
+        writer.println("        <div class='container' style='font-size:large;'>");
+        writer.println("            <table class='table table-bordered' style='width:100%;text-align:center;'>");
+        writer.println("                <thead>");
+        writer.print("                      <tr class='" + overallCssClass + "'>");
+        headerCell("Tests");
+        headerCell("Passed");
+        headerCell("Skipped");
+        headerCell("Failed");
+        headerCell("Time (ms)");
+        writer.println("</tr>");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "</tr>" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)